include/zephyr/drivers/can/transceiver.h

```diff
@@ -41,9 +41,16 @@ typedef int (*can_transceiver_enable_t)(const struct device *dev, can_mode_t mod
  */
 typedef int (*can_transceiver_disable_t)(const struct device *dev);
 
+/**
+ * @brief Callback API upon setting CAN transceiver termination
+ * See @a can_transceiver_term() for argument description
+ */
+typedef int (*can_transceiver_term_t)(const struct device *dev, bool state);
```

Comment:
should this be something like `temination_enable` and `termination_disable`? for symmetry with the two above

Reply:
Could be done, but ends up with one function more in the API. Except the "cosmetic" symmetry would be here another reason for it?

Reply:
done